### PR TITLE
Manage Cafcass Access

### PR DESCRIPTION
**Pull request by sahaSonali**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `enable_keep_helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPET-1058

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/ManageOrdersController.java [hunk 1]
```diff
@@ -165,6 +168,14 @@ public AboutToStartOrSubmitCallbackResponse populateHeader(
             caseDataUpdated.put(CASE_TYPE_OF_APPLICATION, CaseUtils.getCaseTypeOfApplication(caseData));
             if (C100_CASE_TYPE.equalsIgnoreCase(CaseUtils.getCaseTypeOfApplication(caseData))) {
                 caseDataUpdated.put(PrlAppsConstants.CAFCASS_OR_CYMRU_NEED_TO_PROVIDE_REPORT, Yes);
+                caseDataUpdated.put(
```
Comment (shashisk97) [inline src/main/java/uk/gov/hmcts/reform/prl/controllers/ManageOrdersController.java:171]: Why is this change needed?

File: src/main/java/uk/gov/hmcts/reform/prl/enums/ServeOrderFieldsEnum.java [hunk 2]
```diff
@@ -3,14 +3,12 @@
 public enum ServeOrderFieldsEnum {
 
     serveOrderAdditionalDocuments("serveOrderAdditionalDocuments"),
-    isCafcass("isCafcass"),
```
Comment (shashisk97) [inline src/main/java/uk/gov/hmcts/reform/prl/enums/ServeOrderFieldsEnum.java:6]: isCafcass also needs to be cleared?

Comment (sahaSonali) [reply src/main/java/uk/gov/hmcts/reform/prl/enums/ServeOrderFieldsEnum.java:6]: It was cleared earlier, no need to clear now
